_organizers/hama-barhamou.md

```diff
@@ -0,0 +1,9 @@
+---
+github: HamaBarhamou
+layout: base
+name: ISSAKA HAMA Barhamou
+photo_url: /static/img/organizers/barhamou.jpg
+role: member of the Website Committee
```

Comment:
```suggestion
role: Website team
```

Reply:
ok well collected. I take note of the recommendations